### ISSUE TITLE
Suggestion: Monorepo

### ISSUE THREAD

**Issue by lveillard** [Blazity/next-enterprise#78]
First of all, great job!

Would be amazing to have this as a boilerplate (or an equivalent) adapted to a monorepo.

Tailwind configuration, absolute paths… there are a lot of things that require hours of configuration. Mixing the good picks of this boilerplate with a monorepo structure.

**Comment by bmstefanski** [comment]
Hi, thanks!

That was actually a first plan, using a Turborepo and splitting all configuration, UI packages, and stuff like that into separate packages/apps. But I ditched that idea since I figured that not all projects require monorepo and many will find that confusing/with overly increased complexity. 

Monorepo would make a lot of sense if we decided to add a backend server to it, then we could create a package for shared types/interfaces etc. But for most projects I think, having monorepo where there is only a single `app` would be more tiring than helpful.

I prefer taking the easier route when it comes to technology, but I am *truly* open to suggestions regarding this because I spent lots of time thinking whether to include this or not, so if you have some pros/cons that I skipped I would love to discuss them 🖖🏻
